godot-core/src/builtin/script.rs

```diff
@@ -0,0 +1,875 @@
+/*
```

Comment:
This file should be called `script_instance.rs`.

It should also live in `godot::engine` module, instead of `godot::builtin`, because it's really not a builtin/core type. At some point we need to organize `godot::engine` a bit better to keep generated and manually written types apart, but that's for another time...